src/main/java/graphql/analysis/NodeVisitorWithTypeTracking.java

```diff
@@ -151,7 +153,12 @@ public TraversalControl visitField(Field field, TraverserContext<Node> context)
         boolean isTypeNameIntrospectionField = fieldDefinition == schema.getIntrospectionTypenameFieldDefinition();
         GraphQLFieldsContainer fieldsContainer = !isTypeNameIntrospectionField ? (GraphQLFieldsContainer) unwrapAll(parentEnv.getOutputType()) : null;
         GraphQLCodeRegistry codeRegistry = schema.getCodeRegistry();
-        Map<String, Object> argumentValues = ValuesResolver.getArgumentValues(codeRegistry, fieldDefinition.getArguments(), field.getArguments(), CoercedVariables.of(variables));
+        Map<String, Object> argumentValues = ValuesResolver.getArgumentValues(codeRegistry,
+                fieldDefinition.getArguments(),
+                field.getArguments(),
+                CoercedVariables.of(variables),
+                GraphQLContext.getDefault(),
+                Locale.getDefault());
         QueryVisitorFieldEnvironment environment = new QueryVisitorFieldEnvironmentImpl(isTypeNameIntrospectionField,
```

Comment:
This entry point come not from a graphql request and hence uses a default graphql context and default locale.  Perhaps in some future change these are passed in.